src/github/utils/config.ts

```diff
@@ -5,7 +5,7 @@ import { Config, configSchema } from "../types/config";
 import { expressionRegex } from "../types/plugin";
 import { eventNames } from "../types/webhook-events";
 
-const UBIQUIBOT_CONFIG_FULL_PATH = ".github/ubiquibot-config.yml";
+const UBIQUIBOT_CONFIG_FULL_PATH = ".github/.ubiquibot-config.yml";
```

Comment:
I expect that there should also be an organization config path as well.

Reply:
By organization config you mean related to the organization where this is installed or our template? I have to add `@ubiquibot/configuration` defaults there as well.